src/AmqpConsumer.php

```diff
@@ -101,9 +101,11 @@ public function setupQueues(): void
         }
     }
 
-    private function acknowledge(AMQPMessage $message): void
+    private function acknowledgeIfRequired(AMQPMessage $message): void
     {
-        $message->getChannel()->basic_ack($message->getDeliveryTag());
+        if (! $this->getConsumerProperty('enable_manual_acknowledgement')) {
```

Comment:
@ethanol-fossil Hi! Thank you for the PR. I think that would be good to add a default value in the next way

```$this->getConsumerProperty('enable_manual_acknowledgement', false)```.